Scripts/Python/xLinkingBookGUIPopup.py

```diff
@@ -527,9 +527,8 @@ def IShowBookNoTreasure(self):
                 gLinkingBook = ptBook(bookdef,self.key)
                 gLinkingBook.setSize( width, height )
                 # make sure there is a cover to show
-                if not showOpen:
-                    if not self.IsThereACover(bookdef):
-                        showOpen = 1
+                if showOpen and self.IsThereACover(bookdef):
+                    showOpen = 0
```

Comment:
This doesn't feel right to me.... Whether or not a book should start open is hardwired to the following "logic":

```python
        if len(actBookshelf.value) == 0:
            agePanel = TargetAge.value
            showOpen = 1  # start pedestal books open
        else:
            agePanel = stringAgeRequested
            showOpen = 0    # start bookself books closed
  ```

Cyan seems to think anything we just find on a pedestal out in the world is "open" by definiton, but anything on a shelf is closed. That means these lines I'm commenting on *should* just be a safety valve for not showing a cover that doesn't exist, which would make this:

```suggestion
                if showOpen and not self.IsThereACover(bookdef):
                    showOpen = 0
```